src/GEM_u8g2.h

```diff
@@ -43,8 +43,8 @@
 #include "constants.h"
 
 // Macro constants (aliases) for u8g2 font families used to draw menu
-#define GEM_FONT_BIG        u8g2_font_6x12_tr
-#define GEM_FONT_SMALL      u8g2_font_tom_thumb_4x6_tr
+#define GEM_FONT_BIG        u8g2_font_6x12_tf
+#define GEM_FONT_SMALL      u8g2_font_tom_thumb_4x6_tf
```

Comment:
Since we are not able to use the 'ä' in names is it still required to change the font, or doesn't it matter anyway?

Reply:
We use a "³" in the main menu dashboard (for a test) - so we need it for now